README.md

```diff
@@ -41,7 +41,7 @@ the same IP address, use a combination of a script, the address from an
 Internet-facing interface, or default to using the IP address change
 detector of the DDNS provider.
 
-__  
+__
```

Comment:
Here we actually need the double space, it's a hint to Markdown to not break the row.  We want the output to look like this:

```
___
¹ ....
```